backend/src/org/commcare/suite/model/Detail.java

```diff
@@ -65,6 +65,10 @@
     // Force the activity that is showing this detail to show itself in landscape view only
     private boolean forceLandscapeView;
 
+    // If this detail is being shown in an entity list, set focus to the LAST item in the list
+    // when it is loaded, instead of the first
+    private boolean focusToBottomOfEntityList;
```

Comment:
I'm not entirely sure why any of the suite stuff is externalizable. We re-parse it from xml every time the app launches.